cernopendata/modules/fixtures/data/docs/delphi-guide-docker/delphi-guide-docker.md

```diff
@@ -20,31 +20,80 @@ xhost + local:docker
+The home directory of the delphi user contains a set of basic examples which can be used as templates.
+### Event simulation
+As an example, let's generate some events with the Pythia generator, and pass them through the DELPHI detector simulation and reconstruction. Todo that, inside the container, first switch to
+the examples folder
 
 ```
 cd examples/pythia
-pythia.sh
+```
+Take a look at the script ```pythia.sh``` in that folder. In the first part, the configuration (also called the title card) for the generator is created and stored in a file named pythia.tit. Here, you can define the desired end states etc. Then, the generator binary is compiled using the provided Makefile. When the generator is run, it will store the generated events in a file called pythia.fadgen.
+
+The last step consist in passing these generated events through the DELPHI detector simulation, reconstruction and short DST creation. The ```runsim``` script takes care of this. The requested detector setup is for the year 1994, using the latest processing version number. The laboratory is set to CERN. This setting is used to setup random number seeds only. The run nummber is set to 1000, and the beam energy to 45.625 which should match the settings in the generator. The gext version instructs the tools to read the external file pythia.fadgen which was just created.
+
+Run the script by typing inside the pythia folder in the container via
+
+```
+./pythia.sh
+```
+The script will create a bunch of files:
+
+* simana.fadsim is the raw simulated data
+* simana.fadana is the reconstructed simulated data
+* simana.sdst is the short DST data which is what should be used for analysis.
+
+### Scanning the short dst events
+
+To run the event display, let's first make sure that the required folders are present. So please run
+```
 mkdir -p ~/graexe/data ~/graexe/hcopy  ~/graexe/macro  ~/graexe/run
+```
+
+For convenience, let's copy over the short dst file
+
+```
 cp simana.sdst ~/graexe/data
+```
+
+Finally, start the event display from the home directory
+```
+cd
 rungra
 ```
+
+It should start up showing an hour glass, and a file selection box, like this:
+
+<p><center><img src="/static/docs/delphi-data-screenshots-2024/delgra_startup_1.png" width="60%"></center></p>
+
+Press on OK, which will bring up the file dialog where you can select the folder and the file to be read in:
+
+<p><center><img src="/static/docs/delphi-data-screenshots-2024/delgra_startup_2.png" width="60%"></center></p>
+
+Switch to ~/graexe/data and select the simana.sdst file which we just copied there, and press Ok in that dialog:
+
+<p><center><img src="/static/docs/delphi-data-screenshots-2024/delgra_startup_3.png" width="60%"></center></p>
+
+When asked which event to read, just press OK to start reading the first event from the run, or press return:
+
+<p><center><img src="/static/docs/delphi-data-screenshots-2024/delgra_startup_3.png" width="60%"></center></p>
```

Comment:
Note that the last three pictures all refer to the same image 3. One of them could refer to image 4, but what about the other? (There are five placeholders, but only four images.) Perhaps one is missing?

![Screenshot 2024-08-12 at 13-07-55 Running the DELPHI software stack using Docker CERN Open Data Portal](https://github.com/user-attachments/assets/98f0f382-3ccd-41fa-942e-9dcb6972bd44)

Reply:
Regarding image 4 that is not linked:

![delgra_startup_4](https://github.com/user-attachments/assets/c9837753-2b9f-43bd-8a14-b33d41db1cf4)

The event is really tiny. Perhaps you could use a bit of zoom there. Motivation: the hour glass is much more prominent, so people can even miss the event there.  If the real event could be displayed of about the  same size as the hour glass, then people will more easily notice the change from one image to another.

Reply:
P.S. Changed last image to 4 during rebase.

Reply:
Hi, 
I have updated the text a bit. There is indeed one reference too much, and the last one was not linked but should have been. 
For the event, the first one in this run indeed does not look terribly exciting ... it may be better to select a different one from that set ... or use a different end state. Will check that.
.